std/algorithm/comparison.d

```diff
@@ -925,7 +925,7 @@ range of range (of range...) comparisons.
 
 @safe pure unittest
 {
-    import std.utf : byChar, byWchar, byDchar;
+    import std.utf : byChar, byDchar, byWchar;
```

Comment:
@andralex if we want to enforce sortedness for all imports, we have to look over a few minor exceptions where the order makes a bit of sense.